src/classes/class/Class.js

```diff
@@ -0,0 +1,46 @@
+import {useParams} from "react-router-dom";
+import CLASSES from "../example";
+import {useEffect, useState} from "react";
+import PersonCard from "./PersonCard";
+
+function Class() {
+  const {classURL} = useParams();
+  let classData = CLASSES.find(c => c.url === classURL);
```

Comment:
Doesnt seem to be a check for when classData is undefined i.e. where the classURL is not found. This can lead to runtime errors, if a incorrect param is passed.